src/components/Ui/Calendar.vue

```diff
@@ -61,23 +64,19 @@ function toggleDay(year, month, day) {
   input.value = formatDate(year, month, day);
   emit('update:modelValue', input.value);
 }
-function isSelectable() {
-  return true;
-  /*
-      const in30Days = new Date();
-      in30Days.setDate(in30Days.getDate() + 30);
-      return (
-        new Date(year, month, day) > new Date() &&
-        new Date(year, month, day) < in30Days
-      );
-      */
+function isSelectable(year, month, day) {
+  const dateForCheck = new Date(year, month, day);
+  const diff = new Date().setHours(0, 0, 0, 0) - dateForCheck;
```

Comment:
cant we use date-fns or moment to do these calculations? I find this strategy prone to mistakes. @bonustrack

Reply:
@midgerate can you provide an example on how that would be possible?

Reply:
@sarmatdev no mistake. We are contemplating to use Day.js its not a blocker though.